awsrds/rds_db_instance_test.go

```diff
@@ -1502,8 +1502,9 @@ var _ = Describe("RDS DB Instance", func() {
 					EngineVersion: aws.String("11"),
 				},
 			})
-			_, err := rdsDBInstance.GetFullValidTargetVersion("postgres", currentVersion, targetVersion)
-			Expect(err).To(HaveOccurred())
+			version, err := rdsDBInstance.GetFullValidTargetVersion("postgres", currentVersion, targetVersion)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(version).To(Equal(""))
```

Comment:
Are we confident that the RDS API returns no results when looking for an upgrade from , say, 11 to 11? If the database was 11.X could it return 11.Y?